Payload_Type/thanatos/thanatos/mythic/agent_functions/netstat.py

```diff
@@ -0,0 +1,41 @@
+from mythic_container.MythicCommandBase import (
+    BrowserScript,
+    TaskArguments,
+    CommandBase,
+    CommandAttributes,
+    SupportedOS,
+    MythicTask,
+    PTTaskMessageAllData,
+    PTTaskProcessResponseMessageResponse,
+)
+
+
+class NetstatArguments(TaskArguments):
+    def __init__(self, command_line, **kwargs):
+        super().__init__(command_line, **kwargs)
+        self.args = []
+
+    async def parse_arguments(self):
+        pass
+
+
+class NetstatCommand(CommandBase):
+    cmd = "netstat"
+    needs_admin = False
+    help_cmd = "netstat"
+    description = "Get all active network connections & sockets"
+    version = 1
+    author = "@maclarel"
+    argument_class = NetstatArguments
+    attackmapping = [""]
```

Comment:
The MITRE ATT&CK technique ID (T1049) should be added here to match the [documentation](https://github.com/MythicAgents/thanatos/pull/40/files#diff-f33e632bbffa94f15b5c7491ec653626b7cd6ed070b79b32b6cd6f9c9da666e2R53). This will also allow Mythic to track this TTP being used.
```py
attackmapping = ["T1049"]
```